autokeras/nodes.py

```diff
@@ -96,4 +96,42 @@ def config_from_adapter(self, adapter):
 
 
 class TimeSeriesInput(Input):
-    pass
+    """Input node for Time Series data.
+
+    # Arguments
+        lookback: Int. The range of history steps to consider for each prediction.
+            For example, if lookback=n, the data in the range of [i - n, i - 1]
+            is used to predict the value of step i. If unspecified, it will be tuned
+            automatically.
+    """
```

Comment:
We need the docstrings for column_names and column_types here.

Reply:
Added, but do we really need column_names and column_types? because after teh data transform the "columns" in the transformed dataset is a vector instead of the original single value.

Reply:
Yes, every arg should be accessible even after use.